workflow/scripts/oncoprint.py

```diff
@@ -96,19 +123,27 @@ def variant_oncoprint(gene_calls):
     if len(matrix.columns) > 1:
         # sort by recurrence
         matrix = sort_by_recurrence(matrix, lambda matrix: matrix.isna())
+
+    matrix = attach_group_annotation(matrix, group_annotation)
+
     return matrix.reset_index()
 
+
 calls = pd.concat(
     [
         load_calls(path, sample)
         for path, sample in zip(snakemake.input, snakemake.params.groups)
```

Comment:
This could be what's currently making it fail:

```suggestion
        for path, sample in zip(snakemake.input.calls, snakemake.params.groups)
```